app/scripts/services/pmx-converter.js

```diff
@@ -10,20 +10,20 @@
   function PMXConverter(jsyaml) {
 
     function convert(pmxYaml) {
-      var pmxJson, images, newJson;
+      var pmxJson,
+        images,
+        newJson = {};
 
       try {
         pmxJson = jsyaml.safeLoad(pmxYaml);
-      } catch (YamlException) {
-        return null;
+        images = pmxJson.images;
+        images.forEach(function (image) {
+          newJson[image.name.replace(' ', '')] = imageToServiceDefinition(image);
```

Comment:
Got it.  Apparently the global replace is not supported in chrome https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace